extensions/security-webauthn/runtime/src/main/resources/webauthn.js

```diff
@@ -0,0 +1,241 @@
+"use strict";
```

Comment:
Perhaps you should just include this from maven?

https://repo1.maven.org/maven2/io/vertx/vertx-auth-webauthn/4.2.5/vertx-auth-webauthn-4.2.5-client.js

This will be updated once the new Level 3 spec is released as we hope that the API will handle JSON directly instead of this dance of encode/decode buffers.

Plus we need to support https://github.com/vert-x3/vertx-auth/issues/539 and you will start drifting from the original script and have double maintenance

Reply:
I have updated it with more methods, to be able to only do the first half and get the login/registration challenges without calling the callback.